content.php

```diff
@@ -10,40 +10,56 @@
     $imageAttributes = wp_get_attachment_image_src( get_post_thumbnail_id(), 'full' ); // returns an array
     
     if ( $imageAttributes ) {
-        $articleStyle = 'style="background-image: url( ' . $imageAttributes[0] . ' )"';
+        $articleStyle = 'style="background-image: background: linear-gradient(45deg, rgba(0,0,0,0.3) 0%,rgba(0,0,0,0.1) 48%,rgba(0,0,0,0) 100%), url( ' . $imageAttributes[0] . ' )"';
+        
     }
 }
 
 ?>
-<article id="post-<?php the_ID(); ?>" <?php post_class(); ?> <?php echo $articleStyle; ?>>
+<style>
+.site-main article.post-<?php the_ID(); ?> {
+    background: linear-gradient(45deg, rgba(0,0,0,0.3) 0%,rgba(0,0,0,0.1) 48%,rgba(0,0,0,0) 100%), url( <?php echo $imageAttributes[0]; ?> );
```

Comment:
Add a check for `! empty( $imageAttributes )` so that this won't give an error when there's no image